package/MDAnalysis/topology/LAMMPSParser.py

```diff
@@ -401,7 +404,8 @@ def _parse_vel(self, datalines, order):
 
         return vel
 
-    def _parse_bond_section(self, datalines, nentries, mapping):
+    def _parse_bond_section(self, datalines: List,
```

Comment:
I think `datalines` could probably be relaxed to `Sequence` here, judging by the requirements of the simple function code.